go/libkb/globals.go

```diff
@@ -391,7 +391,7 @@ func (g *GlobalContext) LogoutUsernameWithSecretKill(mctx MetaContext, username
 	return nil
 }
 
-func (g *GlobalContext) ConfigureLogging() error {
+func (g *GlobalContext) ConfigureLogging(usage *Usage) error {
```

Comment:
Might be worth a quick function comment to say when `usage` can be `nil`.

Reply:
ok, I will do this in the next PR to not re trigger CI - thanks for looking!